crates/qt-build-utils/src/lib.rs

```diff
@@ -489,6 +489,10 @@ impl QtBuild {
         };
 
         for qt_module in &self.qt_modules {
+            if qt_module.contains("Private") {
```

Comment:
Could you add a comment explaining how Private does not have a separate library and is just the normal library name.

I assume there is not a usecase of only putting like CorePrivate and *not* Core ? As this code currently requires you to specify the modules Core *and* CorePrivate, is this the same behaviour as with CMake ?